static/script.js

```diff
@@ -1,183 +1,220 @@
-        var blob = new Blob([child.querySelector('#input-body').value], { type: 'text/plain' });
-        var newFile = new File([blob], child.querySelector('#input-name').value, { type: 'text/xml' });
-      }else{
-        var blob = new Blob([child.querySelector('#input-body').value], { type: 'text/plain' });
-        var newFile = new File([blob], child.querySelector('#input-name').value, { type: 'text/plain' });
-      }
-      uploadedFiles.push(newFile);
+pageSetup();
+
+function pageSetup() {
+    const dropArea = document.getElementById('drop-area');
+
+    document.addEventListener('dragover', function(e) {
+        e.preventDefault();
+    });
+
+    document.addEventListener('drop', function(e) {
+        e.preventDefault();
+    });
+
+    ['dragenter', 'dragover'].forEach(eventName => {
+        dropArea.addEventListener(eventName, highlight, false)
+    });
+
+    ['dragleave', 'drop'].forEach(eventName => {
+        dropArea.addEventListener(eventName, unhighlight, false)
+    });
+
+    ['dragenter', 'dragover', 'dragleave', 'drop'].forEach(eventName => {
+        dropArea.addEventListener(eventName, preventDefaults, false)
+    });
+
+    dropArea.addEventListener('drop', handleDrop, false);
+
+    document.getElementById('file-upload').addEventListener('change', function(event) {
+        let file_arr = [];
+        for (let i = 0; i < event.target.files.length; i++) {
+            file_arr.push(event.target.files[i]);
+        }
+        file_arr.forEach(addFile);
+        event.target.value = "";
+        setFileDividers(document.getElementById('file-container'));
+    });
+
+    document.getElementById('reset-files-button').addEventListener('click', function(event) {
+        clearFiles();
+        showUserInfo("");
+    });
+
+    document.getElementById('submit-button').addEventListener('click', function(event) {
+        let filesContent = getFilesContent();
+
+        if (filesContent.length <= 0) {
+            showUserInfo("Please add a file first");
+            return;
+        }
+
+        let formData = new FormData();
+        filesContent.forEach(element => {
+            formData.append('files[]', element);
+        });
+
+        fetch('/multipart', {
+                method: 'POST',
+                body: formData,
+            })
+            .then(
+                response => {
+                    if (response.ok) {
+                        downloadURL(response.url, "nm-migrated.tar")
+                    } else {
+                        response.text().then(body => showUserInfo(body)).catch(e => showUserInfo(e));
+                    }
+                }
+            ).catch(error => {
+                showUserInfo("Network error occurred. Please try again.");
+            });
+
+        showUserInfo("");
+    });
+}
+
+function showUserInfo(text) {
+    const userInfo = document.getElementById('user-info');
+    userInfo.textContent = text;
```

Comment:
I'm not really sure what you mean?

Reply:
A dialog which looks like a popup: https://jqueryui.com/dialog/#modal-confirmation

Reply:
The idea behind this `user-info` is that later when #8 gets implemented that things like warnings etc. are also supposed to show up here and I think having this as a popup would not work as well.

Reply:
:thinking: not sure why this will not work. But lets not stop this PR with such thing. If I think it is worth to improve here, I will dig in myself.

=> Lets leave it as it is